client/src/components/FullLogo/Logo.stories.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import Logo from './Logo';
+import TagLine from './TagLine'
+import LogoTag from './LogoTag'
+
+storiesOf('Logo', module)
+    .add('Logo', () => <Logo />)
+    .add('TagLine', () => <TagLine />)
+    .add('LogoTag', () => <LogoTag />)
```

Comment:
This is cool. Can you simplify it by making all this a single component that takes a prop to determine if the tagline is visible? We want to so something like `<Logo tagline />` or just `<Logo />`